### PR TITLE
Use dev tools script, codeception removal

### PR DESCRIPTION
**Pull request by mattbell87**
Patch CI to use dev tools script

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -16,7 +16,7 @@ env:
   DB_PASSWORD: cmfive_test
   DB_DATABASE: cmfive_test
   PLAYWRIGHT_MODULES: "admin channel form tag report task timelog"
-  BOILERPLATE_IMAGE: ghcr.io/2pisoftware/cmfive:develop
+  BOILERPLATE_IMAGE: ghcr.io/2pisoftware/cmfive:pr-119 # TODO: change back to develop
```
Comment (mattbell87) [inline .github/workflows/ci.yml:19]: Change to develop

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -58,7 +58,7 @@ jobs:
         uses: actions/checkout@v4
         with:
           repository: '2pisoftware/cmfive-boilerplate'
-          ref: 'feat/phpunit-10' #TODO: change back to develop
+          ref: 'feat/phpunit-10' # TODO: change back to develop
```
Comment (mattbell87) [inline .github/workflows/ci.yml:61]: Change to develop

**Comment by mattbell87** [comment]
Will change branch back to develop before merge

**Comment by mattbell87** [comment]
Ready to be merged. 

Merge at the same time as https://github.com/2pisoftware/cmfive-boilerplate/pull/119

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -118,10 +118,11 @@ jobs:
             -v ${{ github.workspace }}/boilerplate/test:/var/www/html/test:rw \
             -v ${{ github.workspace }}/boilerplate/storage:/var/www/html/storage:rw \
             -e DB_HOST=mysql-8 \
-            -e DB_USERNAME=cmfive_test \
-            -e DB_PASSWORD=cmfive_test \
-            -e DB_DATABASE=cmfive_test \
+            -e DB_USERNAME=$DB_USERNAME \
```
Comment (chris-bateman) [inline .github/workflows/ci.yml:121]: Where are these set?

Comment (mattbell87) [reply .github/workflows/ci.yml:121]: They're set here

https://github.com/2pisoftware/cmfive-core/blob/ba711f473e4f04ea0236b775b26e7744c248b368/.github/workflows/ci.yml#L15-L17

**Review by chris-bateman — APPROVED** [review]

**Pull request merged**